xen-bugtool

```diff
@@ -1516,6 +1520,38 @@ def multipathd_topology(cap):
 
     return stdout
 
+def snmp_xs_conf(cap):
```

Comment:
I would like comments what the replacements achieve. I assume they mask sensitive information by replacing the part to the right of `=`.  The filtering could have been implemented in Python (not using `sed` but using `with file` and `for line in f`) to avoid complicated quoting but I think it's ok for this use case.